### PR TITLE
optimize query

### PR DESCRIPTION
**Pull request by yanns**
@hajoeichler @emmenko please review

### CODE REVIEW

**Comment by hajoeichler** [comment]
Mr optimize ;)

File: src/coffee/service/availability.coffee [hunk 1]
```diff
@@ -79,10 +79,16 @@ module.exports = class
 
   expandVariants: (variants) ->
     if @withInventoryCheck
-      ie = @client.inventoryEntries.all().whereOperator('or')
+
+      skus = ""
       _.each variants, (v) ->
-        ie.where("sku = \"#{v.sku}\"") if v.sku
-      ie.fetch()
+        skus += "\"#{v.sku}\"" if v.sku
```
Comment (emmenko) [inline src/coffee/service/availability.coffee:85]: Shouldn't `skus` be an array?

Comment (yanns) [inline src/coffee/service/availability.coffee:85]: yes, error from me

Comment (emmenko) [inline src/coffee/service/availability.coffee:85]: :+1:

**Pull request merged**